filters/filter_20_Annoyances_MobileApp/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 20,
+  "name": "AdGuard Mobile App Banners filter",
+  "description": "Blocks irritating banners that promote mobile apps of websites. Already included in AdGuard Annoyances filter.",
+  "timeAdded": 1657180419636,
+  "homepage": "https://kb.adguard.com/general/adguard-ad-filters",
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 4,
+  "subscriptionUrl": "https://filters.adtidy.org/extension/chromium/filters/20.txt",
+  "tags": [
+    "purpose:annoyances",
```

Comment:
purpose:mobile maybe?

Reply:
we have `platform:mobile`

Reply:
Hm, we don't have it, but I can add it in this pr.

Reply:
but it is not only about mobile, sometimes mobile app banners appears in desktop versions of websites and we block them too (or we used to block them)

Reply:
Okay guys, nvm

Reply:
> sometimes mobile app banners appears in desktop versions of websites and we block them too (or we used to block them)

yep, that's true

Reply:
There are a lot of the rules for desktop.